lib/src/super_sliver_list.dart

```diff
@@ -249,6 +254,11 @@ class ListController extends ChangeNotifier {
     super.dispose();
   }
 
+  /// Keeps tarck of created [AnimateToItem] so we could later dispose
```

Comment:
Should it be `track` rather than `tarck`?

```suggestion
  /// Keeps track of created [AnimateToItem] so we could later dispose
```